projects/igniteui-angular/src/lib/drop-down/drop-down-item.component.ts

```diff
@@ -61,18 +97,49 @@ export class IgxDropDownItemBase {
 
     /**
      * Sets/gets if the given item is header
+     * ```typescript
+     *  // get
+     *  let mySelectedItem = this.dropdown.selectedItem();
```

Comment:
There's a slight typo here - IgxDropDownBase.selectedItem is a getter, no need for ();